flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/IOMetrics.java

```diff
@@ -53,6 +53,22 @@ public IOMetrics(Meter recordsIn, Meter recordsOut, Meter bytesLocalIn, Meter by
 		this.numBytesOutPerSecond = bytesOut.getRate();
 	}
 
+	public IOMetrics(
+			int numBytesInLocal, int numBytesInRemote, int numBytesOut, int numRecordsIn, int numRecordsOut,
+			double numBytesInLocalPerSecond, double numBytesInRemotePerSecond, double numBytesOutPerSecond,
+			double numRecordsInPerSecond, double numRecordsOutPerSecond) {
```

Comment:
well there *is* a pattern: L1 are counts, L2 are byte rates, L3 are record rates.

Can change it though.